projects/ui/src/Styles/colors.ts

```diff
@@ -63,6 +63,7 @@ const colorMap = {
   royalPurple: "#7e4bc6",
 
   lightRed: "#fff7f7",
+  lightRedDark10: Color("#fff7f7").darken(0.1).desaturate(0.25).hex(),
```

Comment:
I really do like this way of tweaking colors; makes it easy to change if needed, and is also descriptive of what the color change is meant to do

Reply:
One of my refactor ideas here (probably will do in a follow-up) is making this more variablized, something like:

```ts
const baseColors = {
  red: "#fff7f7",
}
export const colors = {
  lightRedDark10: Color(baseColors.red).darken(0.1).desaturate(0.25).hex(),
...
```

Probably will update the names to be better instead of `lightRedDark10` 😄 , and then convert all the buttons to be mantine buttons that use the mantine theme colors.